src/test/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDbStreamTestRunner.java

```diff
@@ -121,48 +121,48 @@ void stop() {
     void await(int timeoutSeconds) {
         if (streamWorker != null) {
             streamWorker.await(timeoutSeconds);
-            List<MTRecord> recordsReceived = new ArrayList<>(streamWorker.getRecordsReceived());
-            assertEquals(expectedMTRecords.size(), recordsReceived.size(),
-                recordsReceived.size() + " of " + expectedMTRecords.size() + " records received");
-            for (MTRecord recordReceived : recordsReceived) {
-                assertMTRecord(recordReceived, expectedMTRecords);
+            List<MtRecord> recordsReceived = new ArrayList<>(streamWorker.getRecordsReceived());
+            assertEquals(expectedMtRecords.size(), recordsReceived.size(),
+                recordsReceived.size() + " of " + expectedMtRecords.size() + " records received");
+            for (MtAmazonDynamoDb.MtRecord recordReceived : recordsReceived) {
```

Comment:
why is this expanded to explicitly denote the outer class (i.e., "MtAmazonDynamoDb.MtRecord" instead of just "MtRecord")?